test/subscribe.test.js

```diff
@@ -0,0 +1,89 @@
+describe('Subscribe', function() {
+  describe('PersistedModel', function() {
+    describe('configured to source changes locally', function() {
+      before(function() {
+        var test = this;
+        var app = loopback({localRegistry: true});
+        var ds = app.dataSource('ds', {connector: 'memory'});
+        this.Score = app.model('Score', {
+          dataSource: 'ds',
+          changeDataSource: false // use only local observers
```

Comment:
Could you please point me to the code which handles `changeDataSource`? I'd like to understand better the consequences of setting this flag to `false`.

Perhaps you should add unit-tests for `changeDataSource:true` too?